src/modules/Extension/Service.php

```diff
@@ -341,7 +341,7 @@ public function update(\Model_Extension $model)
                 throw new \Box_Exception('Could not retrieve :ext information', [':ext' => $model->name], 744);
             }
 
-            throw new \Box_Exception('Visit the extension store for more information on updating this extension.', null, 252);
+            throw new \Box_Exception('Visit the extension directory for more information on updating this extension.', null, 252);
```

Comment:
The `testupdate` function in `Box\Mod\Extension\ServiceTest` needs to be updated with the new exception wording the fix the test error: 

>>> Failed asserting that exception message 'Visit the extension directory for more information on updating this extension.' contains 'Visit the extension store for more information on updating this extension.'.